events.py

```diff
@@ -1 +1,4 @@
 CLICK = 'click'
+HOVER = 'hover'
+TOOLTIP = 'tooltip'
```

Comment:
This should be removed as I think `TOOLTIP` will eventually fall under `HOVER` anyways. That being said, it's currently not being used, but will be when I use them as constants for the `Button` class.